.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: swift
+xcode_workspace: MiniApp.xcworkspace
+xcode_scheme: Tests
+osx_image: xcode10.2
+
+before_install:
+- bundle install
+- gem update fastlane cocoapods --no-document
+- pod repo update
+
+script:
+- fastlane ci
+
+deploy:
+  provider: pages
```

Comment:
Is GitHub Pages enabled yet?

Reply:
> Is GitHub Pages enabled yet?

No. I couldn't find the settings to enable it. May be I don't have privileges to enable it.

Reply:
Github pages is automatically enabled when you create a branch named `gh-pages`.

Reply:
Yeah. Got it. Will remove these pages deploy config for now and will add them later once we have created one.